src/app/game/services/game.service.ts

```diff
@@ -22,7 +22,8 @@ export class GameService {
   }
 
   getAll(): Observable<Game[]> {
-    return this.httpClient.get<Game[]>(`${this.API_URL}/games`);
+    return this.httpClient.get<Game[]>(`${this.API_URL}/games`)
+      .pipe(map(datas => datas.map(data => createGameLight(data))
```

Comment:
```suggestion
      .pipe(map(datas => datas.map(createGameLight)
```
voir si ca marche comme ça